docker/debug-pod/Dockerfile

```
@@ -30,6 +31,13 @@ RUN ARCH=$(dpkg --print-architecture) && \
     sudo mv etcd-v3.5.0-linux-${ARCH}/etcdctl /usr/local/bin/ && \
     rm -rf etcd-v3.5.0-linux-${ARCH}*
 
+# Kafka Install
+RUN KAFKA_VERSION=3.5.0 && \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)